generate-stackbrew-library.sh

```diff
@@ -12,9 +12,7 @@ declare -A debianSuites=(
 )
 defaultAlpineVersion='3.15'
 declare -A alpineVersions=(
-	# /usr/src/php/ext/openssl/openssl.c:551:12: error: static declaration of 'RSA_set0_key' follows non-static declaration
-	# https://github.com/docker-library/php/pull/702#issuecomment-413341743
-	#[7.0]='3.7'
+	#
```

Comment:
We should leave _a_ comment here (even if it's a bogus example) to remind us how to use this array: :sweat_smile:

```suggestion
	#[8.1]='3.15'
```

Reply:
Yes, that might be a good thing!